GliaWidgets/Glia.swift

```diff
@@ -285,4 +285,18 @@ public class Glia {
         }
         environment.coreSdk.updateVisitorInfo(info, completion)
     }
+
+    /// Ends active engagement if existing and closes Widgets SDK UI (includes bubble).
+    public func endEngagement(_ completion: @escaping (Result<Void, SalemoveError>) -> Void) {
+
+        interactor?.endSession(
+            success: {
+                completion(.success(()))
+            }, failure: {
+                completion(.failure($0))
+            }
+        )
+        onEvent?(.ended)
```

Comment:
While Interactor needs to be reworked in order to be mocked (this will be address some other time), for  now we can make sure that onEvent?(.ended) is called using unit test. Also in test we can make sure that nil is set to rootCoordinator (it has to become `internal` instead of `private`).

Reply:
fixed 8743899cc91de4628974e20a922b41759ba1590c